templating/twig_extension.rst

```diff
@@ -62,9 +62,32 @@ As an example you'll create a price filter to format a given number into price::
     versions before 1.26, include this method which is omitted in the example
     above.
 
+Here's how to create a custom **function**::
+
+    // src/AppBundle/Twig/AppExtension.php
+    namespace AppBundle\Twig;
+
+    use Twig\Extension\AbstractExtension;
+    use Twig\TwigFunction;
+
+    class AppExtension extends AbstractExtension
+    {
+        public function getFunctions()
+        {
+            return array(
+                new TwigFunction('total', array($this, 'totalFunction')),
+            );
+        }
+
+        public function totalFunction(float $price, int $quantity)
```

Comment:
Don't know what you mean exactly. See above line:
> new TwigFunction('total', array($this, 'totalFunction')),

Do you mean it should be changed there too?

Reply:
Yes please, sorry for not being that clear, but Function suffix is not necessary at all

Reply:
Yeah, sure. But since the corresponding filter function (above) also carries the useless "Filter" suffix (`priceFilter()`) at https://symfony.com/doc/2.8/templating/twig_extension.html#create-the-extension-class I figured to go along with it. In the Twig docs they're recommending neither: https://twig.symfony.com/doc/2.x/advanced.html#id2

So I'm fine to delete it, but in this case the "Filter" suffix should probably be deleted too, shouldn't it?

Reply:
Yes, we could do some renaming here:

`priceFilter()` -> `formatPrice()`
`totalFunction()` -> `getTotal()` or `calculateTotal()`

Reply:
👍 for `formatPrice()` and `calculateTotal()`

Reply:
Sure :-)
https://github.com/symfony/symfony-docs/pull/10593